assets/js/modules/analytics/common/index.js

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-export { default as AccountCreate } from './account-create-legacy';
+export { default as AccountCreate } from './account-create';
```

Comment:
As discussed the other day, we will need to include both components here, and use the legacy one if the site is not using the proxy. Let's add a selector `usingProxy` to the `core/site` store for that purpose, and set it based on whether the `_googlesitekitBaseData.proxySetupURL` value exists or not.

Reply:
will do.

Reply:
Added this in https://github.com/google/site-kit-wp/pull/1389/commits/4fd7501de62b7bb620282230bfcf14268ef3177a.